apps/barista/src/components/live-example/live-example.ts

```diff
@@ -153,4 +208,59 @@ export class BaLiveExample implements OnDestroy {
       );
     }
   }
+
+  /**
+   * Creates a non-visible textarea with the given string as value.
+   */
+  private _createTextArea(value: string): HTMLTextAreaElement {
```

Comment:
May you move this into a pure function? Better testable and it needs no class context